### PR TITLE
Base code

### PR DESCRIPTION
**Pull request by jscotka**

### CODE REVIEW

File: conu.spec [hunk 1]
```diff
@@ -0,0 +1,43 @@
+%global framework_name conu
```
Comment (phracek) [inline conu.spec]: I would prefer to move spec into directory rpm.

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+setup(
+    name='conu',
+    version="0.0.1",
+    description='Container testing library',
+    keywords='containers,testing',
+    author='Jan Scotka',
+    author_email='jscotka@redhat.com',
+    url='https://github.com/fedora-modularity/conu',
+    license='GPLv2+',
```
Comment (phracek) [inline setup.py]: License should be GPLv3.

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    packages=find_packages(exclude=['examples']),
+    include_package_data=True,
+    data_files=data_files.items(),
+    entry_points={},
+    setup_requires=[],
+    classifiers=[
+        'Development Status :: 4 - Beta',
+        'Environment :: Console',
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: GNU General Public License v2 or later (GPLv2+)',
```
Comment (phracek) [inline setup.py]: The same here.

File: conu.spec (hunk 1 above)
Comment (jscotka) [reply conu.spec]: Then we'll have same issue  as in MTF, `tito` will not work well and `COPR` will have troubles with using `scm` builds

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,19 @@
+language: python
+python:
+  - "2.7"
```
Comment (phracek) [inline .travis.yml]: We would like to have python 3. and not Python 2.

Comment (jscotka) [reply .travis.yml]: we can change it in future, I need to see that it works well and I know that it woks with python2 now (it is connected with installation via setup.py actually it is done under python2). You can change it then if you know how. or better to say, install it to both of them python2.*/site-packages and python3.*/site-packages

File: setup.py (hunk 2 above)
Comment (jscotka) [reply setup.py]: We can change it after some discussion, for me it is not anyhow important.

File: setup.py (hunk 3 above)
Comment (jscotka) [reply setup.py]: We can change it after some discussion, for me it is not anyhow important.

**Comment by phracek** [comment]
I have nothing against to approve it, but let's wait for the others.

**Comment by jscotka** [comment]
@phracek I think that others wait for this initial code to work on top of it. than discuss inside this initial code.

File: conu.spec (hunk 1 above)
Comment (TomasTomecek) [reply conu.spec]: tito? please don't

File: setup.py (hunk 3 above)
Comment (TomasTomecek) [reply setup.py]: +1

we can have separate discussion about license (I use MIT for my projects, just for record)

**Pull request merged**